### DISCUSSION TITLE
Support description(hints) for variables

### DISCUSSION THREAD

**Discussion by gilkgit** [grafana/grafana#37031] [Dashboards, Panels & Visualizations]
Support description(hints) for variables -
Use case:
I have a dashboard with multiple variable dropdowns.
Some of the users of this dashboard are struggling to understand what the drop-downs are doing.
It would have been helpful if I could add description for each variable which will be presented upon hovering over the variable name.

**Comment by sakjur** [comment]
Please write a description and let me know so I can reopen and categorize this.

**Comment by gilkgit** [comment]
> Please write a description and let me know so I can reopen and categorize this.
@sakjur - Added description.
Perfect - this goes to show how important description is (which I can't add to the dashboard variables)
:)

**Comment by sakjur** [comment]
> Perfect - this goes to show how important description is (which I can't add to the dashboard variables)

:joy: Haha, yes! Thank you for the expansion.

**Note:**
I think this would be an interesting task for someone in the community, but make sure to reach out to us before so we can discuss details a little.

**Comment by hugohaggmark** [comment]
This feature has been available since https://github.com/grafana/grafana/pull/29332 which was released in Grafana 7.4.0-beta1.
